### PR TITLE
Chimera LS refactor and Configurator support

### PR DESCRIPTION
**Pull request by noroadsleft**
- Matrix/keymap refactor
  - Deleted `KC_KEYMAP` macro
  - Renamed `KEYMAP` macro to `LAYOUT_ortho_4x12`
  - Default keymap updated to current QMK standards
    - #include QMK_KEYBOARD_H
    - Refactored from `KC_KEYMAP` to `LAYOUT_ortho_4x12` macro
    - `action_get_macro()` deprecated in favor of `process_record_user()`
  - `rules.mk` **not** updated to include `LAYOUTS = ortho_4x12` because I have no idea if and how that works on a split and/or wireless keyboard.

- readme cleanup: markdown formatting

- Add Configurator support

### CODE REVIEW

**Comment by drashna** [comment]
LAYOUTS works just fine on the rules.mk.  Check out the lets split. https://github.com/qmk/qmk_firmware/blob/master/keyboards/lets_split/rules.mk#L75

File: keyboards/chimera_ls/chimera_ls.h [hunk 1]
```diff
@@ -50,32 +50,20 @@
...
-) \
-{ \
-	{ KC_##k43, KC_##k45, KC_##k34, KC_##k11, KC_##k23, KC_##k40, KC_##k38, KC_##k25, KC_##k00, KC_##k12 }, \
-	{ KC_##k31, KC_##k44, KC_##k46, KC_##k35, KC_##k22, KC_##k28, KC_##k39, KC_##k37, KC_##k24, KC_##k13 }, \
-	{ KC_##k30, KC_##k32, KC_##k33, KC_##k47, KC_##k09, KC_##k29, KC_##k27, KC_##k26, KC_##k36, KC_##k02 }, \
-	{ KC_##k19, KC_##k20, KC_##k21, KC_##k42, KC_##k18, KC_##k16, KC_##k15, KC_##k14, KC_##k41, KC_##k17 },  \
-	{ KC_##k06, KC_##k07, KC_##k08, KC_##k10, KC_NO,    KC_##k05, KC_##k04, KC_##k03, KC_##k01, KC_NO    }, \
-}
+// readability
+#define ___ KC_NO
```
Comment (mechmerlin) [inline keyboards/chimera_ls/chimera_ls.h]: this is potentially confusing when KC_TRNS is seven spaces. `KC_NO` makes it very clear that no means no lol.

Comment (drashna) [reply keyboards/chimera_ls/chimera_ls.h]: I'm not sure I agree with that.  This is in the keyboard's h file, and ... most likely, most users aren't going to see this.

**Comment by drashna** [comment]
Thos was good to go, right?

**Comment by noroadsleft** [comment]
It was back in August. Should be good again now.

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**